src/core/request/deltafetch.js

```diff
@@ -46,132 +42,68 @@ export class DeltaFetchRequest extends KinveyRequest {
 
   execute() {
     return repositoryProvider.getOfflineRepository()
-      .then(repo => repo.read(this._getCollectionFromUrl(), this.query))
-      .catch((error) => {
-        if (!(error instanceof NotFoundError)) {
-          throw error;
+      .then((repo) => {
+        const collectionName = this._getCollectionFromUrl();
+        let queryCacheQuery = new Query()
+          .equalTo('collectionName', collectionName)
+          .descending('lastRequest');
+
+        if (this.query) {
+          queryCacheQuery = queryCacheQuery
+            .and()
+            .equalTo('query', this.query.toString());
```

Comment:
Wouldn't this include `skip` and `limit`? I thought it was decided these will not be part of the persisted query representation. What about sort - should that be part of it? Or maybe just filter, cause I don't think the order of the returned entities matters for our purposes, does it?

Reply:
Yes I should be removing `skip` and `limit`. I am not sure about `sort`. I will just use `filter` for now.